src/utils/wallet.js

```diff
@@ -451,237 +421,25 @@ const wallet = {
-      config.setWalletServiceBaseUrl(walletServiceBaseUrl || WALLET_SERVICE_MAINNET_BASE_URL);
-      config.setWalletServiceBaseWsUrl(walletServiceWsUrl || WALLET_SERVICE_MAINNET_BASE_WS_URL);
-
-      const walletConfig = {
-        seed: words,
-        xpriv,
-        xpub,
-        requestPassword: async () => new Promise((resolve) => {
-          /**
-           * Lock screen will call `resolve` with the pin screen after validation
-           */
-          routerHistory.push('/locked/');
-          store.dispatch(lockWalletForResult(resolve));
-        }),
-        passphrase,
-        network,
-      };
-
-      wallet = new HathorWalletServiceWallet(walletConfig);
-      connection = wallet.conn;
-    } else {
-      // If we've lost redux data, we could not properly stop the wallet object
-      // then we don't know if we've cleaned up the wallet data in the storage
-      // If it's fromXpriv, then we can't clean access data because we need that
-      oldWalletUtil.cleanLoadedData({ cleanAccessData: !fromXpriv});
-
-      let xpriv = null;
-
-      if (fromXpriv) {
-        xpriv = oldWalletUtil.getXprivKey(pin);
-      }
-
-      connection = new Connection({
-        network: network.name,
-        servers: [helpers.getServerURL()],
-      });
-
-      const beforeReloadCallback = () => {
-        store.dispatch(loadingAddresses(true));
-      };
-
-      const walletConfig = {
-        seed: words,
-        xpriv,
-        xpub,
-        store: STORE,
-        passphrase,
-        connection,
-        beforeReloadCallback,
-      };
-
-      wallet = new HathorWallet(walletConfig);
-    }
-
-    store.dispatch(setWallet(wallet));
-
-    // TODO should we save server info?
-    //store.dispatch(setServerInfo(serverInfo));
-    wallet.on('state', async (state) => {
-      if (state === HathorWallet.ERROR) {
-        // ERROR
-        // TODO Should we show an error screen and try to restart the wallet?
-      } else if (wallet.isReady()) {
-        // READY
-        const { tokensHistory, tokensBalance, tokens } = await this.fetchWalletData(wallet);
-        store.dispatch(loadWalletSuccess(tokensHistory, tokensBalance, tokens));
-      }
-    });
-
-    wallet.on('new-tx', async (tx) => {
-      let message = '';
-      if (helpers.isBlock(tx)) {
-        message = 'You\'ve found a new block! Click to open it.';
-      } else {
-        message = 'You\'ve received a new transaction! Click to open it.'
-      }
-
-      const notification = this.sendNotification(message);
-
-      // Set the notification click, in case we have sent one
-      if (notification !== undefined) {
-        notification.onclick = () => {
-          routerHistory.push(`/transaction/${tx.tx_id}/`);
-        }
-      }
-
-      this.fetchNewTxTokenBalance(wallet, tx).then(async (updatedBalanceMap) => {
-        if (updatedBalanceMap) {
-          this.handlePartialUpdate(wallet, updatedBalanceMap);
```

Comment:
Can we remove this `handlePartialUpdate` method?

Reply:
Yes we can, done in 54e0653e27af8005ff919e0825c78d3110e68015